config/application.rb

```diff
@@ -37,6 +37,12 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     config.i18n.default_locale = :de
+    config.middleware.insert_before 0, Rack::Cors do
```

Comment:
warum CORS?  läuft das nicht auf der selben domain?

Reply:
Wenn wir in Richtung einer ReactNative Anwendung gehen wollen, brauchen wir das auf jeden Fall, oder? Ich hab aber auch local schon Probleme, wenn ich ne SPA auf nem anderen Port dagegen laufen lasse